.npmrc

```diff
@@ -4,3 +4,4 @@
 # This file sets the default parameters for manual `pnpm install`.
 #
 shamefully-hoist=true
+strict-peer-dependencies=false
```

Comment:
Why this?

Reply:
`mvn clean package -Pproduction -Dpnpm.enable=true` adds this, should we commit?

Reply:
Right. It is needed for pnpm 7... let's add it then